pkg/extension/grpc/v1/kuadrant.proto

```
@@ -24,8 +24,12 @@ service ExtensionService {
   rpc RegisterMutator(RegisterMutatorRequest) returns (google.protobuf.Empty) {}
   // Clear all subscriptions and registered mutators for a policy
   rpc ClearPolicy(ClearPolicyRequest) returns (ClearPolicyResponse) {}
-  // Register an upstream gRPC service for a policy
-  rpc RegisterUpstreamMethod(RegisterUpstreamMethodRequest) returns (google.protobuf.Empty) {}
+  // Register a named gRPC action method for a policy
+  rpc RegisterActionMethod(RegisterActionMethodRequest) returns (google.protobuf.Empty) {}
```

Buf (1.69.0)

[error] cannot find `google.protobuf.Empty` in this scope

(COMPILE)